functions/go/templater/README.md

```diff
@@ -0,0 +1,60 @@
+# Templater function
+
+This is an example of implementing a templater function.
+Templater allows generating documents using go-template
+engine and Sprig library.
```

Comment:
right naming is always a question :)
Sprig is a name of the library. Template syntax is from go-template. I'm not sure that sprig-template will resonate.
Looks like the function you found can only generate nginx service and deployment templates and used just as an example to demonstrate what functions can do. The image name for that function is:

```
gcr.io/kustomize-functions/example-nginx
```

(see [here](https://github.com/kubernetes-sigs/kustomize/blob/master/functions/examples/template-go-nginx/local-resource/example-use.yaml#L11)), even though it's located in `template-go-nginx` directory.
Templater function is more universal, it can simulate `example-nginx` behavior and generate the same yamls if the appropriate template is included.

I can rename if you insist, but I'm not sure we should. Just let me know if you still fill that you need that

Reply:
Thanks the templater name makes sense based on your explanation.